libs.autobuild

```diff
@@ -20,6 +20,15 @@ cmake_package "drivers/glfw" do |pkg|
 end
 remove_from_default 'drivers/glfw'
 
+import_package 'drivers/imu_myahrs_plus' do |pkg|
+    pkg.post_import do
+        Autobuild::Subprocess.run(pkg, 'build', 'make', :working_directory => pkg.srcdir + "/common_cpp")
```

Comment:
Use `File.join` to join paths. And use key: value for keyword arguments, e.g. `working_directory: File.join(pkg.srcdir, 'common_cpp')`

Moreover, it's best to use pkg.run instead of accessing Subprocess.run directlry, e.g. `pkg.run('build',  ...)`

Finally, access the tool names using Autobuild.tool (which makes them configurable), e.g. `Autobuild.tool(:make)`

Aren't there any way to build from outside the build tree ? This is really bad ...